sig-architecture/README.md

```diff
@@ -24,6 +24,22 @@ The Architecture SIG maintains and evolves the design principles of Kubernetes,
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-architecture)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Farchitecture)
 
+## Subprojects
```

Comment:
Actually, I think k8s.io maybe should be contribX and the rest should be owned by the steering committee.

SIG Arch efforts are listed here:
https://github.com/kubernetes/community/blob/master/sig-architecture/charter.md

Many of the items clearly need to be moved so that they can have dedicated owners files.

Reply:
steering committee doesn't yet exist as a formalized concept of a group as this file / generator are concerned, so I'd like to defer moving to it until then